vue3-apollo-quasar/src/components/IssuesPullRequestsCard/IssuesPullRequestsCard.vue

```diff
@@ -23,21 +23,21 @@
             class="text-h6 custom-icon"
             name="svguse:app-icons/closed-pull-request.svg#closed-pull-request"
             v-else-if="
-              cardType === CARD_TYPES.PULL_REQUEST && state === STATES.MERGED
+              cardType === CARD_TYPES.PULL_REQUEST && state === STATES.CLOSED
```

Comment:
May be nit-picking but I think this would look better in singular form in actual usage:
`CARD_TYPE.PULL_REQUEST` and `STATE.CLOSED`